### PR TITLE
IDGEN-42: Added webservices to Idgen module

### PR DESCRIPTION
**Pull request by samuelmale**
Not yet ready for review.

### CODE REVIEW

**Comment by tendomart** [comment]
@samuelmale  let's see if Travis loves the commit.Did every thing build succesfully?

**Comment by samuelmale** [comment]
I have not yet made any code changes. I'm planning of fetching your branch now.

**Comment by tendomart** [comment]
oops okk thot you had fetched last light.

**Comment by tendomart** [comment]
Cool

File: tory [hunk 1]
```diff
@@ -0,0 +1,230 @@
...
+ ctrl-BACKSPACE   ESC-BACKSPACE             Delete word to left of cursor.
+ ctrl-DELETE      ESC-DELETE      ESC-X     Delete word under cursor.
+ ctrl-U           ESC (MS-DOS only)         Delete entire line.
+ UpArrow                          ESC-k     Retrieve previous command line.
+ DownArrow                        ESC-j     Retrieve next command line.
+ TAB                                        Complete filename & cycle.
+ SHIFT-TAB                        ESC-TAB   Complete filename & reverse cycle.
+ ctrl-L                                     Complete filename, list all.
+
+
```
Comment (samuelmale) [inline tory]: @tendomart  watsup with the above file?

**Comment by tendomart** [comment]
@samuelmale  there was a mess in the command you run during push

**Comment by dkayiwa** [comment]
@samuelmale i guess you meant to close this in favour of https://github.com/openmrs/openmrs-module-idgen/pull/74

**Comment by samuelmale** [comment]
True dat @dkayiwa

**Comment by tendomart** [comment]
Thanks @dkayiwa

**Pull request closed without merging**